src/input-capture.c

```diff
@@ -226,8 +254,18 @@ static XdpOptionKey input_capture_create_session_options[] = {
   { "capabilities", G_VARIANT_TYPE_UINT32, validate_capabilities },
 };
 
+
+static gboolean handle_create_draft_session (XdpDbusInputCapture   *object,
+                                         GDBusMethodInvocation *invocation,
+                                         const char            *arg_parent_window,
+                                         GVariant              *arg_options)
+{
+  return FALSE;
+  //Not Sure what happens here
+}
+
 static gboolean
-handle_create_session (XdpDbusInputCapture   *object,
+handle_init_session (XdpDbusInputCapture   *object,
```

Comment:
The function below should call `xdp_dbus_impl_input_capture_call_create_session()` and filter the options using a new `input_capture_init_session_options` array.

Reply:
Where are these defined ?